src/victory-legend/victory-legend.js

```diff
@@ -0,0 +1,186 @@
+/*global document:false */
+
+import React, { PropTypes } from "react";
+import { merge, assign, isEmpty } from "lodash";
+import VictoryLabel from "../victory-label/victory-label";
+import VictoryContainer from "../victory-container/victory-container";
+import Point from "../point/point"; // it should be replaced
+
+const defaultFontStyle = {
+  fontSize: 14,
+  fontFamily: "'Gill Sans', 'Gill Sans MT', 'Ser­avek', 'Trebuchet MS', sans-serif",
+  color: "#252525",
+  backgroundColor: "#d9d9d9",
+  stroke: "transparent"
+};
+
+const defaultLegendStyle = {
+  height: 200,
+  width: 300,
+  padding: 20
+};
+
+const defaultLegendData = [{
+  name: "Series 1",
+  symbol: {
+    type: "circle",
+    style: {
+      fill: "red"
+    }
+  }
+}, {
+  name: "Series 2",
+  symbol: {
+    type: "diamond",
+    style: {
+      fill: "blue"
+    }
+  }
+}];
+
+export default class VictoryLegend extends React.Component {
+  static propTypes = {
+    x: PropTypes.number.isRequired,
+    y: PropTypes.number.isRequired,
+    height: PropTypes.number.isRequired,
+    width: PropTypes.number.isRequired,
+    orientation: PropTypes.string,
+    data: PropTypes.arrayOf(
+      PropTypes.shape({
+        name: PropTypes.string.isRequired,
+        label: PropTypes.object,
+        symbol: PropTypes.object
+      })
+    ),
+    columnItemSpacing: PropTypes.number,
+    rowItemSpacing: PropTypes.number,
+    groupComponent: PropTypes.element,
+    standalone: PropTypes.bool,
+    style: PropTypes.object
```

Comment:
Legends should be able to work with `theme`, but this can be v2 work if you prefer to get something in